istio-proxy/tar-proxy-source.sh

```diff
@@ -11,11 +11,16 @@ fi
 
 #clone dependency source and custom recipes
 if [ ! -d "recipes" ]; then
-  git clone https://github.com/bdecoste/proxy-rpm
-  mv proxy-rpm/proxy/* .
-  rm -rf proxy-rpm
+  git clone https://github.com/openshift-istio/recipes-istio-proxy -b ${PROXY_VERSION}
 fi
 
+#fetch dependency sources
+for filename in recipes/*.sh
```

Comment:
I don't think this will work; recipes dir is now recipes-istio-proxy (unless you change the checkout dir in the clone command above).